### PR TITLE
chapter 6 practic

### PR DESCRIPTION
**Pull request by Dmitriy-Romanchuk**

### CODE REVIEW

File: chapter_6_practic/task2.cpp [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+#include <iostream>
+#include <conio.h>
+
+class toolBooth
+{
+private:
+	unsigned int counter;
+	double revenue;
+public:
+	toolBooth() : counter(0), revenue(0.0)
```
Comment (zefir-o) [inline chapter_6_practic/task2.cpp]: Now it's time to get familiar with modern C++.
Prefer uniform initialization always. https://www.bogotobogo.com/cplusplus/C11/C11_Uniform_initialization.php

File: chapter_6_practic/task1.cpp [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+// value, and then displaying this result
+
+#include <iostream>
+
+class Int
+{
+private:
+	int value;
+public:
+	Int() : value(0)
```
Comment (zefir-o) [inline chapter_6_practic/task1.cpp]: Are you familar with copy constructors?

File: chapter_6_practic/task5.cpp [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+#include <iostream>
+#include <iomanip>
+
+class date
+{
+private:
+	int day;
+	int month;
+	int year;
+	char ch;
```
Comment (zefir-o) [inline chapter_6_practic/task5.cpp]: Why do you need to keep 'ch' as a member?

Comment (zefir-o) [reply chapter_6_practic/task5.cpp]: Every instance of this class will keep additional 'ch' char. Whereas, this member is needed only locally, in the get function.

File: chapter_6_practic/task6.cpp [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+			job_title = etype::accountant;
+			break;
+		case 'e':
+			job_title = etype::executive;
+			break;
+		case 'r':
+			job_title = etype::researcher;
+			break;
+		default:
+			std::cout << "unexpected letter!" << std::endl;
```
Comment (zefir-o) [inline chapter_6_practic/task6.cpp]: What would be the result in case of an unexpected letter?

File: chapter_6_practic/task7.cpp [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+		angle_longitude_2.showdata();
+		angle_latitude_2.showdata();
+		std::cout << "Repeat? y/n: ";
+		std::cin >> ch;
+	} while ('y' == ch);
+
+	return EXIT_SUCCESS;
+}
+
+void angle::setdata()
```
Comment (zefir-o) [inline chapter_6_practic/task7.cpp:71]: why do you only split declaration and definition for member functions but not for constructors as well?

File: chapter_6_practic/task8.cpp [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+#include <iostream>
+
+class counter
+{
+private:
+	static int count;
+	int number;
+public:
+	counter() : number(count)
```
Comment (igor-epam) [inline chapter_6_practic/task8.cpp:21]: Let's talk about the operator's order and how the code could be improved.

File: chapter_6_practic/task6.cpp [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+{
+private:
+	int number;
+	float compensation;
+	date start_date;
+	etype job_title;
+	char title;
+
+public:
+	emploee() : number(0), compensation(0), job_title(etype::manager), title('l')
```
Comment (zefir-o) [inline chapter_6_practic/task6.cpp]: https://en.cppreference.com/w/cpp/utility/optional

File: chapter_6_practic/task2.cpp (hunk 1 above)
Comment (Dmitriy-Romanchuk) [reply chapter_6_practic/task2.cpp]: Fixed
https://github.com/Dmitriy-Romanchuk/master/pull/6/commits/3a244c2cfa768aa01621e213a62942a91d83b5dc

File: chapter_6_practic/task5.cpp (hunk 3 above)
Comment (Dmitriy-Romanchuk) [reply chapter_6_practic/task5.cpp]: Fixed
https://github.com/Dmitriy-Romanchuk/master/pull/6/commits/6ce145bb54e2fbbd03b277137bf879c738f68190

File: chapter_6_practic/task6.cpp (hunk 4 above)
Comment (Dmitriy-Romanchuk) [reply chapter_6_practic/task6.cpp]: Fixed 
https://github.com/Dmitriy-Romanchuk/master/pull/6/commits/15e0f7a145db1613b6687215eda3a97a5a5b3f2d

File: chapter_6_practic/task7.cpp (hunk 5 above)
Comment (Dmitriy-Romanchuk) [reply chapter_6_practic/task7.cpp:71]: I'm not sure what I did here is right
https://github.com/Dmitriy-Romanchuk/master/pull/6/commits/e232442248b735ef6314d0f0e52618f28806e8ca

File: chapter_6_practic/task6.cpp (hunk 7 above)
Comment (Dmitriy-Romanchuk) [reply chapter_6_practic/task6.cpp]: needs to be discussed. can't understand the syntax

File: chapter_6_practic/task1.cpp (hunk 2 above)
Comment (Dmitriy-Romanchuk) [reply chapter_6_practic/task1.cpp]: needs to be discussed

**Pull request merged**